weWatch/ClientApp/src/components/404.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+// import { instanceConfig } from '../instance'; // need to add instanceConfig for global config
+
+const Wrapper = styled.div`
+    height: 100%;
+`;
+
+export const PageNotFound: FC = () => {
+
+
```

Comment:
unnecessary enters

Reply:
let it be for now, I will clean it in future tasks